### PR TITLE
style(rxBreadcrumb): Update breadcrumbs for 2.0 color palette

### PR DESCRIPTION
**Pull request by glynnis**
https://jira.rax.io/browse/FRMW-335

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

### Before
screen shot 2015-11-16 at 4 45 02 pm
screen shot 2015-11-16 at 4 45 19 pm

On hover:
screen shot 2015-11-16 at 4 46 37 pm

### After (updated 11/19/2015 @ 1:39pm)
screen shot 2015-11-19 at 1 33 19 pm
screen shot 2015-11-19 at 1 33 35 pm

### CODE REVIEW

**Comment by glynnis** [comment]
The contrast ratio on the demo tag even after my changes doesn't pass the contrast ratio requirement from tota11y. However, given that this isn't critical text and that it's much closer to 4.5 (and ≥ 4.0), I feel comfortable with my changes. @ssalinas24 @ElementE What do y'all think?

File: src/styles/vars.less [hunk 1]
```diff
@@ -337,6 +337,5 @@
 /*
  * rxBreadcrumbs
  */
-// TODO: rename with proper component prefix
-@breadcrumbHeight: 1.2em;
-@breadcrumbArrowWidth: 9px;
+@rxBreadcrumbHeight: 1.2em;
+@rxBreadcrumbArrowWidth: 9px;
```
Comment (CITguy) [inline src/styles/vars.less]: If we are to rename these, I'd like to continue with the kebab-ish case seen in other prefixed variable names (lowerCamelCasePrefix-with-kebab-name).

```
@rxBreadcrumbs-height: 1.2em;
@rxBreadcrumbs-arrow-width: 9px;
```

Comment (glynnis) [inline src/styles/vars.less]: Ah, good call! Made the change and pushing up now :+1:

File: src/rxBreadcrumbs/rxBreadcrumbs.less [hunk 2]
```diff
@@ -28,29 +28,29 @@
...
       /* TODO add in border gradient to match nav background */
-      border-left-color: #dddddd;
+      border-left-color: @gray-400;
       z-index: 2;
     }
     a::before {
-      border-left-color: #cccccc;
-      right: -(@breadcrumbArrowWidth + 1);
+      border-left-color: @gray-500;
+      right: -(@rxBreadcrumbArrowWidth + 1);
```
Comment (CITguy) [inline src/rxBreadcrumbs/rxBreadcrumbs.less]: :wink: - variables in this file will also need renamed to reflect the change to `vars.less`

Comment (glynnis) [inline src/rxBreadcrumbs/rxBreadcrumbs.less]: :tired_face: der

**Comment by CITguy** [comment]
LGTM

**Comment by 100stacks** [comment]
LGTM

**Comment by ssalinas24** [comment]
@glynnis and @ElementE When does the gray appear behind the blue Demo Tag? In the current demo I see that the background behind it is transparent at http://rackerlabs.github.io/encore-ui/#/components/rxBreadcrumbs

**Comment by glynnis** [comment]
@ssalinas24 and I just discussed the contrast issue here, and I'm working with @ElementE on getting some darker blues added to the palette. Leaving this open until we see whether a darker blue would help with the contrast ration of the demo tag.

**Comment by ElementE** [comment]
Here is the new value for Blue-900: #0D47A1
blue-900

**Comment by glynnis** [comment]
Updated code and "after" screenshots above.

Here's a screenshot of changes to the color palette page:
screen shot 2015-11-19 at 1 33 03 pm

Tags are now using `@blue-900` for improved contrast ratio.

**Comment by ElementE** [comment]
Design LGTM

**Pull request merged**